File: dev-docs/snippets/cron-triggers/get.swift
import JsBaoClient

// Get a cron trigger by id, including runtime state from the Durable Object.
func get(client: JsBaoClient, triggerId: String) async throws {
  // #region example
  let trigger = try await client.cronTriggers.get(triggerId: triggerId)
  let state = trigger.state
  // #endregion example
  _ = state
}

File: dev-docs/snippets/cron-triggers/get.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Get a cron trigger by id, including runtime state from the Durable Object.
export async function get(client: JsBaoClient, triggerId: string) {
  // #region example
  const trigger = await client.cronTriggers.get(triggerId);
  // #endregion example
  return trigger;
}
